packages/manager/modules/overthebox/src/overTheBox.scss

```diff
@@ -1,6 +1,6 @@
 ovh-manager-over-the-box-component { /* stylelint-disable-line */
```

Comment:
This file should be renamed as `index.scss` in order to keep consistency across modules. Isn't it?

Reply:
yes